varnish-cache/templates/hpa.yaml

```diff
@@ -24,6 +24,8 @@ spec:
     name: {{ include "varnish-cache.fullname" . }}
   minReplicas: {{ .Values.server.autoscaling.minReplicas }}
   maxReplicas: {{ .Values.server.autoscaling.maxReplicas }}
+  behavior:
```

Comment:
Is it possible to guard this with `{{- if not (empty .Values.server.autoscaling.behavior) }}` ?